scripts/deploy-frontend.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/sh
+
+# source deploy-util.sh
+
+# The deployment strategy you wish to employ ( rolling update or setting up a new environment)
+DEPLOY_STRATEGY=${1}
+
+# The environment in which you want to execute these commands
+DEPLOY_ENV=${2}
+
+#The application name  defined via the manifest yml for the frontend
+CGHOSTNAME_FRONTEND=${3}
+CGHOSTNAME_BACKEND=${4}
+
+
+update_frontend()
+{
+
```

Comment:
This script needed to be more involved than the one for the backend. Originally this logic was being done inside of the dockerfile, and was done remotely.

Now we are compiling the app locally before issueing the deployment